WooCommerce/Classes/ViewRelated/Orders/ProductListViewController.swift

```diff
@@ -98,7 +98,8 @@ extension ProductListViewController: UITableViewDelegate {
 
         if FeatureFlag.productDetails.enabled {
             let orderItem = itemAtIndexPath(indexPath)
-            productWasPressed(for: orderItem.productID)
+            let productIDToLoad = orderItem.variationID == 0 ? orderItem.productID : orderItem.variationID
```

Comment:
This looks like a good candidate to be abstracted in a method (maybe an extension on OrderItem)? It is used twice (for now) and it feels like it is an abstraction leaking (we need the UI to be aware of the fact that a product variation is a product but not really... Do you think it makes sense?

Reply:
Yeah, good point @ctarda — I agree. This can be cleaned up in a subsequent PR.